esp8266-fastled-webserver.ino

```diff
@@ -511,6 +514,18 @@ void setup() {
     sendInt(clockBackgroundFade);
   });
 
+  webServer.on("/timeZone", HTTP_POST, []() {
+    String value = webServer.arg("value");
+    setTimeZone(value.toInt());
+    sendInt(currentTimeZoneIndex);
+  });
+
+  webServer.on("/dst", HTTP_POST, []() {
+    String value = webServer.arg("value");
+    setDst(value.toInt());
+    sendInt(dst);
+  });
```

Comment:
Daylight Saving Time is, unfortunately, not a `boolean`.  Different areas of the world start / stop DST at different dates and/or times.

Existing rules that I am aware of include:
0. No DST -- Yay!
1. US_Canada_2021 -- starts at 2AM second Sunday in March, ends 2AM first Sunday in November
2. 2010_Newfoundland -- starst at 12:01AM, ends 12:01AM (same dates as US) ... NO LONGER USED AS OF 2011
3. Mexico_2021 -- starts at 2AM on first Sunday in April, ends 2AM last Sunday in October
4. Cuba_2013 -- Starts at midnight (standard time) ... same dates as US.
5. EU_Britain -- Starts 1AM **GMT** (regardless of timezone) last Sunday in March, Ends last Sunday in October.  `NOTE THE USE OF GMT FOR CALCULATION OF START/STOP`
 
See, for example, http://ontimezone.com/exceptions.php for some differences, or view the fun video on timezones linked above.